docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: "3"
+
+services:
+  emintdnode0:
+    container_name: emintdnode0
+    image: "ethermint/ethermintnode"
+    ports:
+      - "26656-26657:26656-26657"
+      - "1317:1317"
+    environment:
+      - ID=0
+      - LOG=${LOG:-emintd.log}
```

Comment:
Are these env variables used inside the container?

Reply:
I'd need to double-check 👍